src/main/java/com/powsybl/openloadflow/dc/WoodburyEngine.java

```diff
@@ -132,13 +132,13 @@ private Map<PropagatedContingency, WoodburyEngineResult.WoodburyStates> computeS
 
             DenseMatrix flowStatesOverride = flowStates;
             if (rhsModifications.getFlowRhsOverrideByPropagatedContingency(contingency).isPresent()) {
-                flowStatesOverride = rhsModifications.getFlowRhsOverrideByPropagatedContingency(contingency).get();
+                flowStatesOverride = rhsModifications.getFlowRhsOverrideByPropagatedContingency(contingency).orElseThrow();
                 solveRhs(loadFlowContext, flowStatesOverride, reporter);
             }
```

Comment:
FYI, the optional is rather meant to be used in this way
rhsModifications.getFlowRhsOverrideByPropagatedContingency(contingency)
.ifPresent(flowStates -> solveRhs(loadFlowContext, flowStates, reporter));

Reply:
Noted, thank you!